packages/react/src/Dialog/Dialog.tsx

```diff
@@ -28,7 +28,9 @@ const DialogRoot = forwardRef(
   ) => (
     <dialog {...restProps} ref={ref} className={clsx('ams-dialog', className)}>
       <header className="ams-dialog__header">
-        <Heading size="level-4">{heading}</Heading>
+        <Heading level={2} size="level-4">
```

Comment:
[A thread on which heading level to use for modal dialogs](https://github.com/mui/material-ui/issues/34250). I think `h1` makes most sense here.

Reply:
Good find! [This comment](https://github.com/mui/material-ui/issues/34250#issuecomment-1244006106) is indeed convincing.